config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openshift/origin-cluster-nfd-operator
-  newTag: 4.11.0
+  newName: quay.io/eduardoarango/cluster-nfd-operator
```

Comment:
@ArangoGutierrez your private quay repo is listed here, should it be changed to the openshift quay repo ?

Reply:
ups ;P